### ISSUE TITLE
[expr.pre] p3 "operators that are overloaded" is not a defined term

### ISSUE THREAD

**Issue by xmh0511** [cplusplus/draft#6190]
[expr.pre] p3 says
> Subclause [expr.compound] defines the effects of operators when applied to types for which they have not been **overloaded**.

We just say a function operator implements an operator, as per [over.oper.general] p1. We didn't define "an operator is overloaded".  Moreover, [over.match.oper] p1 says
> If no operand of an operator in an expression has a type that is a class or an enumeration, the operator is assumed to be a built-in operator and interpreted according to [expr.compound].

So, [expr.pre] p3 can be changed to
> Subclause [expr.compound] defines the effects of operators when applied to types that are neither class nor enumeration type. 

This is concise and clear.

**Comment by xmh0511** [comment]
> [...] when no operator function is selected.

But, the built-in operators for built-in types are not considered to be operator functions.

**Comment by frederick-vs-ja** [comment]
> But, the built-in operators for built-in types are not considered to be operator functions.

Yes, this is exactly the reason why I suggested the shorter form. I think the sentence should just mean that the effects of built-in operators are covered in [expr.compound].

**Comment by xmh0511** [comment]
> > But, the built-in operators for built-in types are not considered to be operator functions.
> 
> Yes, this is exactly the reason why I suggested the shorter form. I think the sentence should just mean that the effects of built-in operators are covered in [expr.compound].

"when no operator function is selected" could imply this meaning: there is no viable operator function. However, the built-in type does not have operator functions.
